client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
+
+.PinkHeader {
+    background-image: var(--profile-banner);
+    background-repeat:no-repeat;
+    background-size: cover;
+    width: 1440px;
+    height: 231px;
+    text-align: center;
+    padding-top: 3px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+h3{
+    font-family: 'Reem Kufi', sans-serif;
+    color: #FAFAFA;
```

Comment:
Can you use the CSS variable for all colors too

Reply:
yes